MailCore/Models/Attachment.swift

```diff
@@ -53,22 +53,43 @@ public class Attachment: /* Hashable, */ EmbeddedObject, Codable, Identifiable {
 
     public var icon: MailResourcesImages {
         guard let uti else { return MailResourcesAsset.unknownFile }
+
+        if uti.conforms(to: .pdf) {
```

Comment:
Eventualy (not right now), it would be nice to have this abstracted in {core/ui} so we could share this between drive and mail